### PR TITLE
Move test files

### PR DESCRIPTION
**Pull request by c410-f3r**
r? @petrochenkov

### CODE REVIEW

**Comment by petrochenkov** [comment]
issue-41394.rs - enum-discriminant
issue-41394.rs - enum-discriminant
issue-6458* - I think all these tests can be removed now
field-patterns-in-pattern-macro.rs - lint

@rustbot author

**Comment by c410-f3r** [comment]
> issue-41394.rs - enum-discriminant issue-41394.rs - enum-discriminant issue-6458* - I think all these tests can be removed now field-patterns-in-pattern-macro.rs - lint
> 
> @rustbot author

Done

File: tests/ui/inference/issue-6458.rs
Comment (petrochenkov) [inline tests/ui/inference/issue-6458.rs:1]: This test can also be removed.

Comment (c410-f3r) [reply tests/ui/inference/issue-6458.rs:1]: My Bad. Fixed

**Pull request merged**